images/libvirtd/libvirtd-ds.yaml

```diff
@@ -28,9 +28,31 @@ spec:
           - name: LIBVIRTD_DEFAULT_NETWORK_DEVICE
             value: "weave"
           volumeMounts:
-            - mountPath: /host
-              name: host
+            - mountPath: /host-dev
+              name: host-dev
+            - mountPath: /sys
+              name: host-sys
+            - name: libvirt-data
+              mountPath: /var/lib/libvirt
+            - name: libvirt-runtime
+              mountPath: /var/run/libvirt
+        - name: virtlogd
+          image: kubevirt/libvirtd:devel
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: libvirt-runtime
+              mountPath: /var/run/libvirt
+          command: ["/usr/sbin/virtlogd"]
       volumes:
-        - name: host
+        - name: libvirt-data
+          hostPath:
+            path: /var/lib/libvirt-container
```

Comment:
Nack.

So - this would keep the data in the same path as a hosts libvirtd woul dbe. but we should avoid this problem - for several reasons (conflict with host libvirtd, selinux, …).

What I suggest: Let's create /var/lib/libvirtd-ds on the host, and in there use two subpaths: `lib/` and `run/` for the lib and run paths inside the container.